Patch: fix_missing_language_crash.patch
Author: Paul Wise <pabs3@bonedaddy.net>
Fix crash when a language is missing (Closes: #257606)
Index: b/Mailman/htmlformat.py
===================================================================
--- a/Mailman/htmlformat.py	2010-05-24 17:51:04.000000000 +0200
+++ b/Mailman/htmlformat.py	2010-05-24 17:51:08.000000000 +0200
@@ -298,7 +298,7 @@
 
     def Format(self, indent=0, **kws):
         charset = 'us-ascii'
-        if self.language:
+        if self.language and Utils.IsLanguage(self.language):
             charset = Utils.GetCharSet(self.language)
         output = ['Content-Type: text/html; charset=%s\n' % charset]
         if not self.suppress_head:
